cmd/libs/go2idl/deepcopy-gen/generators/deepcopy.go

```diff
@@ -531,6 +535,10 @@ func (g *genDeepCopy) doSlice(t *types.Type, sw *generator.SnippetWriter) {
 }
 
 func (g *genDeepCopy) doStruct(t *types.Type, sw *generator.SnippetWriter) {
+	if len(t.Members) == 0 {
```

Comment:
Probably this is a gap in the snippet writer that we need to figure out a way to do better.